examples/backend-pipeline-pr-checks/Jenkinsfile

```diff
@@ -67,6 +67,15 @@ pipeline {
 
                     }
                 }
+                
+                stage ('Run vulnerability tests') {
+                    if (env.RUN_PLATSEC == true)
+                        steps {
+                            withVault([configuration, vaultSecrets: secrets]) {
+                                sh 'source ${CICD_ROOT}/platsec.sh $IMAGE $ARTIFACTS_DIR' 
```

Comment:
please quote the variables to prevent expansion

```suggestion
                                sh 'source ${CICD_ROOT}/platsec.sh "$IMAGE" "$ARTIFACTS_DIR"' 
```

Reply:
i did it on one line, but added your suggestion